exercise_0/fileMode/FileMode.cpp

```diff
@@ -0,0 +1,22 @@
+//
+// Created by emyrmak on 7/3/2019.
+//
+
+#include "FileMode.h"
+#include <iostream>
+#include <fstream>
+
+int FileMode::run(const string &path) {
+    ifstream  file;
+    file.open(path);
+    if (file) {
+        while(file >> inputText) {
+            characterCounter.checkCharacter(inputText);
+        }
+        characterCounter.getCounts();
+        file.close();
```

Comment:
File is automatically closed at destruction. No need to call `close()` explicitly.